src/components/WebSecurity/SubHeader.jsx

```diff
@@ -48,7 +50,9 @@ const SubHeader = (props) => {
                                       </DropdownUl>
                                   </DropdownContainer>
                               ) : (
-                                  <NavLink to={item.link}>{item.title}</NavLink>
+                                  <NavLink to={item.link} style={{ color: currentSubPath === item.link && "#ff6b08" }}>
```

Comment:
Add default the "Topics" option because it appears first when we click on this page or change the redirection link to open it be `http://localhost:3000/websecurity/topics` and not `http://localhost:3000/websecurity`

![image](https://github.com/thecyberworld/TheCyberHUB/assets/76536506/483bfa97-bc2b-4261-b0a2-c69e48ccd4bd)